src/test/tests/hybrid/movie.py

```diff
@@ -78,8 +77,12 @@ def TestMovieFrames(testFormatString, startindex, framefiles, percents=[], label
         Test(testname)
         DeleteAllPlots()
         CloseDatabase(f)
-    if annot != None:
-        annot.Delete()
+    if len(label) > 0:
+        annotations = GetAnnotationObjectNames()
+        for a in annotations:
+            if a.startswith("Text2D"):
+                annot.Delete()
```

Comment:
Should this be 'a.delete()' ?

Reply:
It should actually by GetAnnotationObject(a).Delete(). Good catch! I don't know how that even worked, since annot in theory would have gone out of scope, assuming Python scoping works like C/C++.
Pushed the fix.

Reply:
wow, good catch!

Looks like it was doing:

`annote.Delete()` followed by `annote.Delete()`   at least one or more times. 

We could make that "ok" (not crash) -- but that is the wrong logic. 

It doesn't surprise me that it could crash.